float32 velocityRatio
float32 accelerationRatio
bool isQueued
---
int32 result
uint64 queuedCmdIndex
